### ISSUE TITLE
Snap fails to launch on Ubuntu 18.10

### ISSUE THREAD

**Issue by n8willis** [h3poteto/whalebird-desktop#926]
## Description
As of the most recent refresh, the snap from snapcraft does not launch on Ubuntu 18.10. Terminal output is the following error:

"[22069:0523/123714.422481:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /snap/whalebird/56/chrome-sandbox is owned by root and has mode 4755."

## How To Reproduce
1. run `whalebird`

**Comment by franksmcb** [comment]
Same issue on Ubuntu MATE 19.04 via snap.

[20394:0523/235413.731688:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /snap/whalebird/56/chrome-sandbox is owned by root and has mode 4755.
Trace/breakpoint trap (core dumped)

**Comment by LeMisterV** [comment]
This is probably linked to this electron issue:
https://github.com/electron/electron/issues/17972

Looks like there's something that can be done during packaging to avoid this error.

I hope this will get fixed easily.

Thanks a lots for the app by the way, really nice 👌

**Comment by LeMisterV** [comment]
By the way, for those who want to run whalebird, you can revert to the previous version installed (if any) using this CLI command:
```sh
snap revert whalebird
```

Those who can't revert (no previous version installed), you should just download the previous version (2.7.1) and install it

**Comment by h3poteto** [comment]
Oh, I have already updated stable, so there isn't previous version's link in snap store. I see, I will release the new version to hot fix, so please wait for a moment.

**Comment by h3poteto** [comment]
I have reverted electron version, and I prepared version 2.7.3 in snap store for release candidate. Could you please try RC version? You can install:
```
$ sudo snap install whalebird --candidate
```

**Comment by h3poteto** [comment]
I released 2.7.3, so please try it. Please tell me if you still can't start it.

**Comment by LeMisterV** [comment]
Works for me, thx @h3poteto !
